### ISSUE TITLE
More established library

### ISSUE THREAD

**Issue by fresheneesz** [cosmicanant/recursive-diff#1]
I'm wondering if you had seen this module: https://github.com/flitbit/diff . It looks like it has very similar algorithm (looks like array changes are taken item by item, rather than attempting to find a more minimal diff - same as flitbit's diff does). Figured you might want to check it out.

**Comment by cosmicanant** [comment]
https://github.com/flitbit/diff may be useful for some users. Also It may have similar algorithm but not exactly same. 
See, I made an attempt towards showing the minimal and relevant information in diff object. Also I tried to write a less complex code with fewer numbers of lines.

I found following things are sufficient enough to explain diff  : 
1. Path : where has the change occurred ! (a '/' separated path just like Unix directories )
2, Operation : update/delete or add
3. value : if added or update, what is new value

Though flitbit/diff is an excellent library, but at the time of writing **recursive-diff**,  I could not found it much useful for the specific scenario I was facing in my project. 
Anyways, choose the library whichever suits your requirement well.
